packages/next/server/image-optimizer.ts

```diff
@@ -15,6 +15,7 @@ import { processBuffer, Operation } from './lib/squoosh/main'
 import Server from './next-server'
 import { sendEtagResponse } from './send-payload'
 import { getContentType, getExtension } from './serve-static'
+import Sharp from 'sharp'
```

Comment:
Should we use `import type` here to make sure we don't accidentally attempt importing `sharp` when it's not available?

Reply:
Event better, on line 33 you can do `let sharp: import('sharp')`